### PR TITLE
Add export to codepen button

### LINKED ISSUE #20: Export to/Open in Codepen Button
**Is your feature request related to a problem? Please describe.**
Would be nice to open what you're working on in codepen

**Describe the solution you'd like**
A export to/open in codepen button

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by SaraVieira**
## Type of Change

- [ ] Bugfix (a fix that does not break existing functionality)
- [x] Enhancement (new feature/minor improvement)
- [ ] Breaking change (fix or feature that would cause existing functionality to break)

## What is Changed

Adds a button that exports the current gradient into a pen

## Comments

closes #20
screenshot 2018-12-06 at 18 25 08

## Checklist

- [x] I have read the **CONTRIBUTING** guideline.
- [x] I have given this pull request a clear and descriptive title
- [x] I have clearly described the changes that this pull request makes
- [x] I have included relevant information or comments

### CODE REVIEW

**Comment by vicbergquist** [comment]
Omg thanks 😍

The height and width of the canvas div should be the values edited by the user, could you please update it to grab those values for the export?

File: src/components/GradientList.vue [hunk 1]
```diff
@@ -45,18 +44,53 @@
...
   computed: {
-    ...mapGetters(['gradientList', 'previewGradient', 'gradientStrings'])
+    ...mapFields(['canvas']),
+    ...mapGetters(['gradientList', 'previewGradient', 'gradientStrings']),
+    getFormData() {
+      return JSON.stringify({
+        title: 'Shapy Gradient 🤖',
+        html: '<div class="gradient"></div>',
+        css: `
+body, html {width: 100%; height: 100%}
```
Comment (SaraVieira) [inline src/components/GradientList.vue]: This needs to be here because the default is 100% by 100% and if I don't set this the gradient won't even show :(

**Comment by SaraVieira** [comment]
@vicbergquist Fixed :)

**Comment by vicbergquist** [comment]
> @vicbergquist Fixed :)

awesome thank you! 🙏

**Pull request merged**